examples/generalized_linear_models/GLM-negative-binomial-regression.myst.md

```diff
@@ -30,10 +32,11 @@ from scipy import stats
 ```
 
 ```{code-cell} ipython3
+warnings.simplefilter(action="ignore", category=FutureWarning)
```

Comment:
It might be possible to avoid ignorning warnings by using the just released seaborn https://github.com/mwaskom/seaborn/releases/tag/v0.13.0rc0.

Reply:
That's right! Thanks for the heads up. I have removed the warning suppression from the notebook.